src/Symfony/Component/Mailer/README.md

```diff
@@ -3,9 +3,66 @@ Mailer Component
 
 The Mailer component helps sending emails.
 
+Getting Started
+---------------
+
+```
+$ composer require symfony/mailer
```

Comment:
Maybe add a comment that using a 3rd party transport like in the example below would require the specific `symfony/*-mailer` package as well?